mwdb/web/src/components/ShowSample.js

```diff
@@ -234,6 +234,21 @@ function SampleDetails() {
     );
 }
 
+function xorArrayBuffer(buffer) {
+    /*
+    xor buffer with key equal 255
```

Comment:
```suggestion
    negate the buffer contents (xor with key equal 0xff)
```
(and chage the name to `negateBuffer`)